src/status_im/contexts/wallet/send/transaction_confirmation/view.cljs

```diff
@@ -275,6 +287,8 @@
              :amount             amount
              :account            account
              :to-address         to-address
+             :type               type
```

Comment:
let's specify the `type` maybe? i.e is it recpient type? we have account, transaction types etc too so let's be sure to be clear about it :)

Reply:
Yes, it is. I will change it to `recipient-type` 👍

Reply:
Done